frontend/public/models/yaml-templates.ts

```diff
@@ -1243,6 +1243,60 @@ metadata:
 spec:
   hrefTemplate: 'https://example.com/logs?resourceName=\${resourceName}&containerName=\${containerName}&resourceNamespace=\${resourceNamespace}&podLabels=\${podLabels}'
   text: Example Logs
+`,
+  )
+  .setIn(
+    [referenceForModel(k8sModels.ConsoleQuickStartModel), 'default'],
+    `
+apiVersion: console.openshift.io/v1
+kind: ConsoleQuickStart
+metadata:
+  name: example
+spec:
+  displayName: Deploy example application from image
+  tags:
+  - example
+  - image
+  - deployment
+  durationMinutes: 5
+  description: Example application.
+  prerequisites:
+  introduction: '### This Quick Start shows you how to deploy an existing image to OpenShift.'
+  tasks:
+    - title: Deploying an existing Image from an Image registry.
+      description: |-
+        ### To deploy the **hello-openshift** image:
+        1. Using the perspective switcher at the top of the navigation, go to **</> Developer**.
+        2. Go to the **+Add** page in the navigation.
+        3. Using the project drop-down list, select the project you would like to deploy the image in.
+        4. Click **Container Image** to deploy an image.
+        5. Type **openshift/hello-openshift** into the **Image name from external registry** field.
```

Comment:
We'll want to use `registry.redhat.io/openshift4/ose-hello-openshift-rhel8`. (Note to QE: This image will only be available after 4.7 GA.)